react/features/recording/hooks.web.ts

```diff
@@ -47,17 +47,14 @@ export function useLiveStreamingButton() {
     const toolbarButtons = useSelector((state: IReduxState) => state['features/toolbox'].toolbarButtons);
     const localParticipantIsModerator = useSelector(isLocalParticipantModerator);
     const liveStreaming = useSelector(getLiveStreaming);
-    const liveStreamingEnabledInJwt
-        = useSelector((state: IReduxState) => isJwtFeatureEnabled(state, 'livestreaming', true));
+    const liveStreamingAllowed = useSelector((state: IReduxState) =>
+        isJwtFeatureEnabled(state, 'livestreaming', localParticipantIsModerator, localParticipantIsModerator));
     const _isInBreakoutRoom = useSelector(isInBreakoutRoom);
 
     if (toolbarButtons?.includes('recording')
-            && isLiveStreamingButtonVisible({
-                localParticipantIsModerator,
-                liveStreamingEnabled: liveStreaming?.enabled,
-                liveStreamingEnabledInJwt,
-                isInBreakoutRoom: _isInBreakoutRoom
-            })) {
+            && !_isInBreakoutRoom
+            && liveStreaming?.enabled
+            && liveStreamingAllowed) {
```

Comment:
Why did we remove the isLiveStreamingButtonVisible function isn't the condition the same here and in the component?

Reply:
Nope, as it depends on the isModerator only if the token is missing or there are no features in the token.